config/environments/production.rb

```diff
@@ -76,4 +76,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Use Mandrill for mail delivery.
+  config.action_mailer.smpt_settings = {
+    address: "smtp.mandrillapp.com",
+    port: 25,
+    enable_starttls_auto: true,
+    user_name:  ENV['MANDRILL_USERNAME'],
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.